src/main/java/com/minecolonies/coremod/entity/ai/citizen/cook/EntityAIWorkCook.java

```diff
@@ -346,46 +448,105 @@ private AIState cookFood()
         }
         walkTo = null;
         setDelay(STANDARD_DELAY);
+        // TODO: why COOK_COOK_FOOD instead of getState()?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)